packages/veritone-react-common/src/components/TranscriptEngineOutput/index.js

```diff
@@ -64,13 +66,17 @@ export default class TranscriptEngineOutput extends Component {
     mediaPlayerTimeMs: number,
     mediaPlayerTimeIntervalMs: number,
     outputNullState: node,
-    bulkEditEnabled: bool
+    bulkEditEnabled: bool,
```

Comment:
it's alarming that there are no required props whatsoever, is this intentional?

Reply:
Added isRequired for the nested shapes, but the upper level peoperties should not be required. Some of those mus bo in pair - if one set - then the other one should be set, but mostly component should show what was passed into it.